spartan/aztec-network/values/alpha-testnet.yaml

```diff
@@ -0,0 +1,78 @@
+telemetry:
+  enabled: true
+
+aztec:
+  realProofs: true
+  numberOfDefaultAccounts: 0
+  testAccounts: true
+  sponsoredFPC: true
+  bootstrapENRs: "enr:-LO4QLbJddVpePYjaiCftOBY-L7O6Mfj_43TAn5Q1Y-5qQ_OWmSFc7bTKWHzw5xmdVIqXUiizum_kIRniXdPnWHHcwEEhWF6dGVjqDAwLTExMTU1MTExLTAwMDAwMDAwLTAtMTgwNmEwMjgtMWE1MzBmM2KCaWSCdjSCaXCEI8nh9YlzZWNwMjU2azGhA-_dX6aFcXP1DLk91negbXL2a0mNYGXH4hrMvb2i92I0g3VkcIKd0A,enr:-LO4QN4WF8kFyV3sQVX0C_y_03Eepxk5Wac70l9QJcIDRYwKS6aRst1YcfbTDdvovXdRfKf-WSXNVWViGLhDA-dUz2MEhWF6dGVjqDAwLTExMTU1MTExLTAwMDAwMDAwLTAtMTgwNmEwMjgtMWE1MzBmM2KCaWSCdjSCaXCEIicTHolzZWNwMjU2azGhAsz7aFFYRnP5xjTux5UW-HyEQcW_EJrZMT1CNm79N4g-g3VkcIKd0A,enr:-LO4QFrGfkRaCk_iFTeUjR5ESwo45Eov9hx_T1-BLdoT-iHzFgCiHMT4V1KBtdFp8D0ajLSe5HcNYrhalmdJXgv6NTUEhWF6dGVjqDAwLTExMTU1MTExLTAwMDAwMDAwLTAtMTgwNmEwMjgtMWE1MzBmM2KCaWSCdjSCaXCEIlICt4lzZWNwMjU2azGhAlC6nKB3iDtRFqWKWqxf_t-P9hc-SZ6VFBJV4y3bTZBQg3VkcIKd0A"
+  contracts:
+    registryAddress: "0x290667b79dd73ce03cd2fc901b905e7f1b001d18"
+    slashFactoryAddress: "0xc5aceadbb626630e854ffd6cb39105e9c541fa34"
+
+network:
+  public: true
+  setupL2Contracts: false
+  p2pBootstrapNodesAsFullPeers: false
+
+blobSink:
+  enabled: true
+
+bot:
+  enabled: false
+
+pxe:
+  enabled: false
+
+faucet:
+  enabled: false
+
+bootNode:
+  enabled: false
+  # unused.
+  externalHost: "http://localhost:8080"
+
+proverNode:
+  l1FixedPriorityFeePerGas: 3
+  l1GasLimitBufferPercentage: 15
+  l1GasPriceMax: 1000
+  resources:
+    requests:
+      cpu: "3"
```

Comment:
3 cores so we use the new 4 core pool. Leaves a core for K8s/Google usage

Reply:
Does Node's `max-old-space-size` get set as part of the parent values file?

Reply:
Nope, will add